Chummer/sheets/Shadowrun 5 set.xslt

```diff
@@ -1005,14 +1036,24 @@
           </xsl:call-template>
         </xsl:if>
 
-        <xsl:if test="gears/gear[iscommlink = 'True']">
+        <xsl:if test="gears/gear[iscommlink = 'True']
+             or armors/armor/gears/gear[iscommlink = 'True']
+             or cyberwares/cyberware/gears/gear[iscommlink = 'True']
+             or cyberwares/cyberware/children/cyberware/gears/gear[iscommlink = 'True']
+             or weapons/weapon/accessories/accessory/gears/gear[iscommlink = 'True']
+             or gears/gear[isprogram = 'True']
+             or armors/armor/gears/gear[isprogram = 'True']
+             or cyberwares/cyberware/gears/gear[isprogram = 'True']
+             or cyberwares/cyberware/children/cyberwear/gears/gear[isprogram = 'True']
+             or weapons/weapon/accessories/accessory/gears/gear[isprogram = 'True']">
```

Comment:
I see what you're trying to do, but these are awkward and also don't pass all use cases. The problem is nesting: cyberware can have many layers of parents (e.g. modular lower limbs attached to modular full limbs att), as can weapons (via underbarrels) and gear (via plugins). These conditions only catch the first layer. If you want to be thorough, you'd want to walk through every single possible descendent nodeset that can lead to a commlink. This should be possible in XSL because it's possible in XPath, I just don't know how off the top of my head.